02_calculator/calculator.rb

```diff
@@ -1 +1,31 @@
-#write your code here
+# write your code here
+def add(num1, num2)
+  num1 + num2
+end
+
+def subtract(num1, num2)
+  num1 - num2
+end
+
+def sum(array)
+  array.inject(0) { |sum, num| sum + num }
```

Comment:
Great use of `.inject`! You could also use `.sum` here as well. Ruby provides lots of convenient enumerables for us